seqr/models.py

```diff
@@ -98,6 +98,8 @@ class Project(ModelWithGUID):
     can_edit_group = models.ForeignKey(Group, related_name='+', on_delete=models.PROTECT)
     can_view_group = models.ForeignKey(Group, related_name='+', on_delete=models.PROTECT)
 
+    genome_version = models.CharField(max_length=5, choices=GENOME_VERSION_CHOICES, default=GENOME_VERSION_GRCh37)
```

Comment:
I like this. Do we want to update the saved variant page to use this instead of the saved_variant_json version, or do we want to do that in a separate PR?

Reply:
whichever you prefer

Reply:
lets do it now so we don't forget